versions.tf

```diff
@@ -2,7 +2,7 @@ terraform {
   required_providers {
     aws = {
       source  = "hashicorp/aws"
-      version = ">= 4.9.0"
+      version = ">= 4.34.0"
```

Comment:
Can you revert this change? In modules the minimum required version should be added which is current 4.9.0. People using the module should have the flexibility to define their own version constraints.